rotkehlchen/data_migrations/migrations/migrations_15.py

```diff
@@ -21,27 +20,46 @@
 def data_migration_15(rotki: 'Rotkehlchen', progress_handler: 'MigrationProgressHandler') -> None:
     """Introduced at v1.34.1
 
-    Sets the protocol for all Hop LP tokens received in an Add liquidity event to 'hop_lp'"""
+    Redecodes the distinct Add liquidity events for each Hop LP token."""
     progress_handler.set_total_steps(1)
 
     with rotki.data.db.conn.read_ctx() as cursor:
-        # find distinct Hop LP token received in an Add liquidity event
-        for identifier, in cursor.execute(
-            'SELECT DISTINCT asset FROM evm_events_info AS EE '
+        # find distinct Add liquidity events for each Hop LP token
+        for db_tx_hash, db_location in cursor.execute(
+            'SELECT DISTINCT tx_hash, location FROM evm_events_info AS EE '
             'LEFT JOIN history_events as HE on HE.identifier=EE.identifier '
-            'WHERE counterparty=? AND type=? AND subtype=?;', (
+            'WHERE counterparty=? AND type=? AND subtype=? GROUP BY asset;', (
                 CPT_HOP,
                 HistoryEventType.RECEIVE.serialize(),
                 HistoryEventSubType.RECEIVE_WRAPPED.serialize(),
             ),
-        ):  # set their protocol to Hop
+        ):
+            tx_hash = deserialize_evm_tx_hash(db_tx_hash)
+            chain_id: SUPPORTED_CHAIN_IDS = ChainID.deserialize(Location.deserialize_from_db(db_location).to_chain_id())
+            chain_manager = rotki.chains_aggregator.get_evm_manager(chain_id=chain_id)
+            with rotki.data.db.user_write() as write_cursor:
+                write_cursor.execute(
```

Comment:
no no no. Don't do that. This is deleting the transaction. And then repulling it. Check `_reset_decoded_events in the last upgrade`. We should not pull anything again. No remote network stuff should happen.